docs/tutorials/lerobot/train_and_evaluate.rst

```diff
@@ -75,34 +120,76 @@ Evaluating Your Policy
 You can use the ``record`` function from :guilabel:`lerobot/scripts/control_robot.py` but with a **policy checkpoint as input**.
 Run the following command to record **10 evaluation episodes**:
 
-.. code-block:: bash
-
-   python lerobot/scripts/control_robot.py \
-     --robot.type=trossen_ai_bimanual \
-     --control.type=record \
-     --control.fps=30 \
-     --control.single_task="Grasp a lego block and put it in the bin." \
-     --control.repo_id=${HF_USER}/eval_act_trossen_ai_bimanual_test \
-     --control.tags='["tutorial"]' \
-     --control.warmup_time_s=5 \
-     --control.episode_time_s=30 \
-     --control.reset_time_s=30 \
-     --control.num_episodes=10 \
-     --control.push_to_hub=true \
-     --control.policy.path=outputs/train/act_trossen_ai_bimanual_test/checkpoints/last/pretrained_model \
-     --control.num_image_writer_processes=1
+.. tabs::
+
+   .. group-tab:: Trossen AI Stationary
+
+      .. code-block:: bash
+
+         python lerobot/scripts/control_robot.py \
+           --robot.type=trossen_ai_stationary \
+           --control.type=record \
+           --control.fps=30 \
+           --control.single_task="Grasp a lego block and put it in the bin." \
```

Comment:
When we recorded, we made this `Test recording episode using Trossen AI Stationary`. This should be similar